app/src/main/java/com/example/myapplication/Main.kt

```diff
@@ -0,0 +1,99 @@
+/**
+ * Функция main для вызова функций всех заданий
+ */
+
+fun main() {
+    task2()
+    task3()
+    task4()
+    task5()
+}
+
+/**
+* Задание 2: “Hello world”
+* Создайте первую программу на Kotlin, которая выводит “Hello, world!” в консоль.
+*/
+
+fun task2() {
+    println("Hello, World!") //Вывод строки в консоль
+}
+
+/**
+ * Задание 3: Работа с переменными
+ * Создайте несколько переменных разных типов (например, Int, String, Boolean).
+ * Присвойте им значения и выведите результаты на экран.
+ * Попробуйте изменить значения этих переменных и снова выведите результаты.
+ */
+
+fun task3() {
+    //создание переменных нескольких типов
```

Comment:
```suggestion
    // создание переменных нескольких типов
```
nit, но в комментариях всегда пробел между // и самой строкой коммента
ну или форматирование IDE делать